low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view

```diff
@@ -867,18 +866,9 @@ class SSPLHealthView(object):
             show_fru = "/show/configuration"
 
         response = self.fetch_ws_request_data(show_fru, fru)
-        if response.status_code != singleton_realstorencl.ws.HTTP_OK:
-            if (response.status_code == singleton_realstorencl.ws.HTTP_TIMEOUT or \
-                response.status_code == singleton_realstorencl.ws.HTTP_CONN_REFUSED or \
-                response.status_code == singleton_realstorencl.ws.HTTP_NO_ROUTE_TO_HOST) \
-                and self.tried_alt_ip is False:
-                print("Retrying to fetch data from another mc IP....: {}".format(singleton_realstorencl.active_ip))
-                singleton_realstorencl.login()
-                self.tried_alt_ip = True
-                response = self.fetch_ws_request_data(show_fru, fru)
-        
-        if response.status_code != singleton_realstorencl.ws.HTTP_OK and self.tried_alt_ip is True:
-            self.correct_mc_config = False
+        if not response or response.status_code != singleton_realstorencl.ws.HTTP_OK:
+            if singleton_realstorencl.active_ip == singleton_realstorencl.mc2:
+                self.correct_mc_config = False
```

Comment:
do w really need this correct_mc_config ? Valid or invalid data response from get_realstor_show_data() would simply can be used to determine whether to proceed with further processing or not, for which correct_mc_config is used today, but i see it unnecessary.

Reply:
**do w really need this correct_mc_config ?**
>> Yes, I have used it to reduce unnecessary call if configuration is not correct. you can check below line code.
https://github.com/Seagate/cortx-monitor/blob/bf3bd584cb8c068e54f2b30585ac7ce1276bd95a/low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view#L180

Reply:
Alright